gradio/interface.py

```diff
@@ -171,12 +171,20 @@ def __init__(self, fn, inputs=None, outputs=None, verbose=False, examples=None,
         self.server_port = server_port
         self.simple_server = None
         self.allow_screenshot = allow_screenshot
-        self.allow_flagging = os.getenv("GRADIO_FLAGGING") or allow_flagging
+        # If parameter is provided, use that; otherwise, environmet variable; otherwise, True
+        if allow_flagging is None:
+            self.allow_flagging = bool(os.getenv("GRADIO_FLAGGING") or True)
```

Comment:
Want to use the same format of os env logic used for theme? 
`self.theme = theme if theme is not None else os.getenv("GRADIO_THEME", "default")`
so
`self.allow_flagging = allow_flagging if allow_flagging is not None else bool(os.getenv("GRADIO_FLAGGING", "1"))`

Reply:
Sure